.travis.yml

```diff
@@ -4,18 +4,25 @@ go:
   - "1.10"
 
 env:
-  - DEP_VERSION="0.4.1"
+  - DEP_VERSION="0.4.1" PROTOC_VERSION="3.6.1"
 
-before_install:
-  # Download the binary to bin folder in $GOPATH
+install:
+  # Download and install dep
   - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
-  # Make the binary executable
   - chmod +x $GOPATH/bin/dep
+  # Download and install protobuf
+  - curl -L -s https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip -o protoc.zip
+  - unzip protoc.zip
+  - cp -r ./bin/protoc ./include $HOME/bin/
+  # Download and install protoc-gen-gogo
+  - go get github.com/gogo/protobuf/protoc-gen-gogo
+  # Download  plugin golang dependencies
+  - make vendor
 
 script:
-  - make vendor
-  - make install
-  # NOTE: We will be able to run protoc commands from Travis once
-  # protoc-gen-atlas-query-validate is added to the atlas gentool. For now, we can just run
-  # tests to ensure that everything works as expected.
-  - go test -v ./...
\ No newline at end of file
+  - make test
+
+# Trigger build/push of latest gentool
+deploy:
+  provider: script
+  script:  curl -LO --retry 3 https://raw.github.com/mernst/plume-lib/master/bin/trigger-travis.sh && sh trigger-travis.sh infobloxopen atlas-gentool $TRAVIS_ACCESS_TOKEN
```

Comment:
If 3-rd party script changes they will be able to run anything they want on our travis. Sounds intriguing!

We can use a version linked to certain commit if we want to be more boring  - https://raw.githubusercontent.com/mernst/plume-lib/a85c9c07207c9001cfbce1afa6f08a0e22071622/bin/trigger-travis.sh